docs/ref/contrib/postgres/operations.txt

```diff
@@ -30,8 +30,8 @@ For example::
             ...
         ]
 
-Django checks that the extension already exists in the database and skips the
-migration if so.
+Django checks if the extension already exists in the database, and if it does,
+it skips adding it again.
```

Comment:
There's a lot of "it" here, referring to the extension, then Django, and finally the extension again. It was a bit confusing to parse. Maybe the second "it" can just be dropped?

Reply:
Yes, one of my worst writing habits. I've rephrased as: "The operation skips adding the extension if it already exists."